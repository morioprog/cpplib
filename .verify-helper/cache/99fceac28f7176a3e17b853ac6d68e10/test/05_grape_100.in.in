100
2 24 40
0
2 48 80
2 41 71
0
0
0
0
2 12 66
0
0
0
0
2 16 76
2 23 99
2 8 89
0
0
2 2 59
2 27 33
1 72
0
2 38 91
0
2 21 31
0
2 60 95
2 25 70
0
2 87 93
0
2 36 61
2 17 45
0
0
0
2 4 64
0
0
0
0
2 22 56
0
0
2 46 92
2 84 98
2 53 67
2 14 65
0
2 75 83
2 30 94
2 69 88
2 32 96
0
0
2 57 82
0
0
2 11 51
0
0
0
0
0
2 19 34
0
2 42 86
2 3 37
0
0
2 68 78
0
0
2 52 97
2 28 49
0
2 7 58
2 20 43
2 15 79
0
2 5 50
0
2 35 74
2 47 62
0
0
2 26 63
2 9 13
2 10 18
0
2 54 77
2 85 90
0
0
2 39 55
2 29 81
0
0
2 1 44
2 6 73
100
1 17
0 55 75
0 13 7
1 24
0 10 62
1 1
0 79 66
1 4
1 28
0 32 28
0 68 26
0 35 19
0 52 9
1 78
0 24 85
0 48 56
0 9 3
0 48 82
1 86
0 60 15
1 44
1 13
0 67 45
1 38
0 52 99
0 30 74
0 66 38
0 21 63
1 75
0 94 9
0 33 98
0 99 22
1 93
1 3
0 30 4
0 81 56
0 91 86
1 28
0 63 15
1 26
1 52
0 7 61
0 92 59
1 91
0 88 38
1 43
1 96
1 5
1 53
1 21
1 80
0 93 97
0 70 23
1 22
0 87 35
1 79
1 45
1 24
0 11 43
0 61 39
0 66 69
0 24 42
0 5 44
0 1 11
0 75 39
0 68 78
0 15 9
1 12
0 16 50
1 58
1 56
1 62
1 69
0 99 63
1 56
0 43 10
1 34
1 58
0 4 37
0 71 45
0 83 28
0 85 23
0 67 79
0 81 5
1 80
0 45 56
0 54 99
1 22
0 4 34
1 61
0 27 33
0 14 69
0 69 54
0 82 73
0 96 7
0 30 87
0 27 42
1 82
1 37
1 77
