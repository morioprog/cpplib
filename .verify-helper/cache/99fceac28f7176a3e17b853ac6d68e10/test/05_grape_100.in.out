0
82
3446
630
3974
2310
4887
9550
6784
11440
15951
10811
18807
20510
11709
22669
27285
28083
26689
22160
27608
1363
22160
29613
10327
29273
1455
15256
25898
35915
32909
26759
38057
7160
27285
35898
52513
4565
51198
60194
61230
